packages/react-components/src/components/styles/VideoTile.styles.ts

```diff
@@ -37,53 +37,66 @@ export const overlayContainerStyles: IStyle = {
   zIndex: 5
 };
 
+/**
+ * @private
+ */
+export const tileInfoContainerStyle = mergeStyles({
+  position: 'absolute',
+  bottom: '0.5rem',
+  left: '0.5rem',
+  minWidth: 'calc(100% - 1rem)',
```

Comment:
is setting minWidth and maxWidth to the same value different than setting just width?

Reply:
I think setting minWidth and maxWidth is much more strict for the children inside. Let me verify.